scripts/apps/search/services/SearchService.js

```diff
@@ -584,6 +584,19 @@ export function SearchService($location, gettext, config, session) {
     };
 
     /**
+     * @ngdoc method
+     * @name doesSearchAgainstRepo
```

Comment:
Hi @akintolga, please check with `grunt docs` that this displays properly. Here, I think you have to use `@name search#doesSearchAgainstRepo` to link it to the service.

I think you'll also have to document SearchService for this to show up, would you mind doing that? Otherwise there's really no point in documenting the methods. Remember that the "@name" for SearchService is "search" as it is defined.